dawn/src/CMakeLists.txt

```diff
@@ -13,15 +13,14 @@
 ##===------------------------------------------------------------------------------------------===##
 
 add_subdirectory(dawn)
-add_subdirectory(dawn-c)
```

Comment:
As far as I can tell, they were created as a public interface to Dawn, since no examples called directly into Dawn. Now that we're publishing a clean interface to calling dawn with a C++ API, there's no use case for the C bindings.

Reply:
conclusion was to remove and build from scratch if needed